source/settings.json

```diff
@@ -381,10 +393,15 @@
       "default": "#FFFBF5"
     },
     {
-      "variable": "text_color",
+      "variable": "primary_text_color",
       "label": "Text",
       "default": "#4E4E4E"
     },
+    {
+      "variable": "link_text_color",
+      "label": "Link text",
```

Comment:
"Link text" sounds a bit awkward for the user-facing label - does "Links" make more sense here?